packages/framework/src/Resources/scripts/admin/plugins/jquery.counterup.js

```diff
@@ -28,7 +28,9 @@
             var num = $this.attr('data-value');
             var nums = [num];
             var isComma = /[0-9]+,[0-9]+/.test(num);
+            var isSpace = /\s/.test(num);
```

Comment:
```suggestion
            var isSpace = /[0-9]+\s+[0-9]+/.test(num);
```

Reply:
I am just figuring out, if the string contains space, so this is not necessary